analyzers/src/SonarAnalyzer.CFG/ShimLayer/OperationInterfaces.xml

```diff
@@ -127,7 +127,7 @@
         </para>
       </summary>
     </Comments>
-    <Property Name="LoopKind" Type="LoopKind">
```

Comment:
Is this file copied unmodified now?

Reply:
The only difference is the lack of `?` in our version of the file. The rest is identical (including formatting inconsistencies) to make maintenance easier.